deploy.sh

```diff
@@ -4,4 +4,4 @@ pip install --upgrade setuptools wheel twine
 rm -rf build/
 rm -rf dist/
 python setup.py sdist bdist_wheel
-twine upload dist/*
+#twine upload dist/*
```

Comment:
Why is this commented out in this PR?

(This PR should be focused on the new parser)